install/python-installer/scripts/traf_sqconfig.py

```diff
@@ -51,18 +52,15 @@ def run():
             lines.append(line)
 
     lines.append('end node\n')
-    lines.append('\n')
-    lines.append('begin overflow\n')
-
-    for scratch_loc in scratch_locs:
-        line = 'hdd %s\n' % scratch_loc
-        lines.append(line)
-
-    lines.append('end overflow\n')
 
+    # write out the node section
     with open(sqconfig_file, 'w') as f:
         f.writelines(lines)
 
+    # append the current persist section to the Trafodion configuration file
+    run_cmd('echo "sed \'1,20d\' %s >> %s"' % (sqconfig_persist_file, sqconfig_file))
+    run_cmd('sed \'1,20d\' %s >> %s' % (sqconfig_persist_file, sqconfig_file))
```

Comment:
why are we stripping header comments out?
seems like we ought to include file verbatim.